root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -328,7 +328,7 @@ const PART_OF_SERIES_LINK_TYPE_GIDS = _.values(PART_OF_SERIES_LINK_TYPES);
 
         linkTypeName() {
             var linkTypeID = this.relationship().linkTypeID();
-            return linkTypeID ? linkPhrase.clean(linkTypeID, this.backward()) : "";
+            return linkTypeID ? linkPhrase.stripAttributes(linkTypeID, this.backward()) : "";
```

Comment:
This throws an exception: `linkPhrase.stripAttributes is not a function`

Reply:
Oh, duh, yeah, this is changed as a separate part of this I haven't yet split and PRd. Will revert for now.